.github/workflows/test.yml

```diff
@@ -0,0 +1,27 @@
+name: Unit tests CI
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
Which ubuntu we are using on our production env? I think we could use Ubuntu 20.04 instead.

Reply:
In case we are using Ubuntu 22.04, we should use `runs-on: ubuntu-22.04` too. We should stick to a version instead of using Github latest version.

Reply:
using `ubuntu-22.04` now